crates/sncast/tests/e2e/account/create.rs

```diff
@@ -73,14 +71,14 @@ pub async fn test_happy_case_generate_salt() {
     ];
 
     let snapbox = runner(&args).current_dir(temp_dir.path());
-    let bdg = snapbox.assert();
-    let out = bdg.get_output();
 
-    let stdout_str =
-        std::str::from_utf8(&out.stdout).expect("failed to convert command output to string");
-    assert!(stdout_str.contains("command: account create"));
-    assert!(stdout_str.contains("max_fee: "));
-    assert!(stdout_str.contains("address: "));
+    snapbox.assert().success().stdout_matches(indoc! {r"
+        command: account create
+        add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
+        address: 0x[..]
+        max_fee: [..]
+        message: Account successfully created[..]
+        "});
```

Comment:
What is the difference between using `assert_stdout_contains` and `.stdout_matches`?

Reply:
`assert_stdout_contains` verifies if the segment is within `stdout`, while `.stdout_matches` ensures that `stdout` contains only the specified segment and nothing else.

Reply:
but in `. stdout_matches` we can use `...`, which skips multiple lines. That allows us to check if the output contains only a segment

Reply:
The following above will fail of only the segment is present:
```rs
snapbox.assert().success().stdout_matches(indoc! {r"
     command: account create
     add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
     address: 0x[..]
     max_fee: [..]
     message: Account successfully created[..]
     ...
     "});
```

Reply:
yeah, my point is that we can do both. We can assert the full output, we can assert a segment with `...`.
but we can say that we assert the full output, but skip some lines with `...`. Although these lines need to exist

Reply:
will also be useful here https://github.com/foundry-rs/starknet-foundry/pull/1708#discussion_r1507356235

Reply:
Sure, will try to have this and #1748 merged to master ASAP